tests/python/unittest/test_numpy_gluon.py

```diff
@@ -602,3 +602,57 @@ def test_pixelshuffle3d():
             [64, 88, 65, 89, 66, 90, 67, 91],
             [68, 92, 69, 93, 70, 94, 71, 95]]]]]
     )
+
+@use_np
+def test_embedding():
+    def check_embedding():
+        layer = gluon.nn.Embedding(10, 100)
+        layer.initialize()
+        x = mx.np.array([3,4,2,0,1])
+        with mx.autograd.record():
+            y = layer(x)
+            y.backward()
+        assert (layer.weight.grad().asnumpy()[:5] == 1).all()
+        assert (layer.weight.grad().asnumpy()[5:] == 0).all()
+
+    def check_embedding_large_input():
+        embedding = mx.gluon.nn.Embedding(10, 1)
+        embedding.initialize()
+        embedding.hybridize()
+        shape = (20481,)
+        with mx.autograd.record():
+            emb_in = embedding(mx.np.ones(shape))
+            loss = emb_in.sum()
+        loss.backward()
+        assert embedding.weight.grad().sum().item() == 20481
+
+    check_embedding()
+    check_embedding_large_input()
+
+@use_np
+def test_layernorm():
+    layer = nn.LayerNorm(in_channels=10)
+    check_layer_forward(layer, (2, 10, 10, 10))
+
+def check_layer_forward(layer, dshape):
```

Comment:
you can use `@pytest.mark.parametrize('arg1,arg2', [(arg1_val, arg2_val)])` to parameterize the test instead of having a wrapper test function.